requirements-cuda.txt

```diff
@@ -9,3 +9,4 @@ torch == 2.4.0
 torchvision == 0.19   # Required for phi3v processor. See https://github.com/pytorch/vision?tab=readme-ov-file#installation for corresponding version
 xformers == 0.0.27.post2  # Requires PyTorch 2.4.0
 vllm-flash-attn == 2.6.1  # Requires PyTorch 2.4.0
+compressed-tensors == 0.4.0
```

Comment:
This needs to be added to requirements-common.txt since compressed-tensors isn't conditionally imported